.circleci/config.yml

```diff
@@ -25,7 +25,7 @@ executors:
   machine-executor:
     working_directory: ~/micrometer
     machine:
-      image: ubuntu-2204:2024.05.1
+      image: ubuntu-2004:2024.08.1
```

Comment:
This looks like an unrelated change. Was this needed for some reason? Note it is downgrading the ubuntu major version from 22.04 to 20.04. If it isn't needed here, let's revert it and upgrade the machine image separately.